tests/pos/AssumedRecursive.hs

```diff
@@ -1,5 +1,6 @@
 module AssumedRecursive where
 
+{-@ lazy foo @-}
 {-@ assume foo :: a -> a @-}
```

Comment:
Ah, so `assume` doesn't disable the termination check anymore. Is this ideal?

Reply:
Exaclty